### PR TITLE
Improve player's turn and win message

### PR DESCRIPTION
**Pull request by slax57**
## Content

- [x] Edit message while in remote play to clarify if it's my turn to play or not
- [x] Improved win message to make it clearer if I won the game or lost it
- [x] Update automated tests
- [x] Use fixed height for messages to avoid screen blinking

Related to: https://trello.com/c/rUzI9XZe/69-05-improve-ui-for-current-player-in-distant-mode

### Win
hex-mobile-remote-win-fixed-height

### Loose
hex-mobile-remote-loose-fixed-height

### CODE REVIEW

File: mobile-app/src/components/Board/Board.tsx [hunk 1]
```diff
@@ -7,23 +7,26 @@ import {
...
+  getPlayersTurnElt,
+  getWinMsgElt,
+}: BoardProps) {
   const svgSize = getApproximateSvgSize(gameState.board.length);
   return gameState.winner ? (
     <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
-      <Text style={{ fontSize: 32 }}>
-        Player {mapStoneColorToPlayerName(gameState.winner)} has won !
-      </Text>
+      <View style={{ height: '100px' }}>{getWinMsgElt(gameState)}</View>
```
Comment (atilbian) [inline mobile-app/src/components/Board/Board.tsx]: I think we can now extract that redundant part and wrap children inside a React Fragment
```tsx
<View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
```

**Comment by djhi** [comment]
A little hover effect would make it easier to use IMO

File: mobile-app/src/components/Board/Board.tsx (hunk 1 above)
Comment (slax57) [reply mobile-app/src/components/Board/Board.tsx]: done

**Comment by atilbian** [comment]
> A little hover effect would make it easier to use IMO

Yes i agree it could be usefull but for dev only, because this app is build as a mobile app only. So i don't know if it's a good idea to add features specifics to web app. I merge this PR as it is, and we can discuss about this later.

**Pull request merged**